sdk/communication/communication-job-router/src/generated/src/jobRouterApiClient.ts

```diff
@@ -54,29 +53,6 @@ export class JobRouterApiClient extends coreClient.ServiceClient {
       baseUri: options.endpoint ?? options.baseUri ?? "{endpoint}"
     };
     super(optionsWithDefaults);
-
-    if (options?.pipeline && options.pipeline.getOrderedPolicies().length > 0) {
```

Comment:
what was all this about?

Reply:
No idea.  Should we raise an issue?

Reply:
@joheredi was this codegen that isn't needed anymore?